nisystemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -25,8 +27,7 @@ def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
         super().__init__(configuration)
 
     @returns.json()
-    @get("nidataframe")
-    # TODO: Create model class for return type
-    def api_info(self) -> dict:
-        """Returns information about API versions and available operations."""
+    @get("nidataframe/v1")
```

Comment:
I changed this method to be version specific. I think it makes more sense to pin the client implementation to a single version of the API. It's also kind of weird to expose the concept of versions when its not relevant to the usage of the client methods (unless we want to explicitly have, for example, `list_tables_v1()`, which I'm not in favor of).

Reply:
Would it make sense to have a constant or something for the "nidataframe/v1" part since every URL will have it?

Reply:
Done